mkdocs.yml

```diff
@@ -627,6 +627,7 @@ nav:
                   - DCIM: "development/apps/migration/model-updates/dcim.md"
                   - Extras: "development/apps/migration/model-updates/extras.md"
                   - IPAM: "development/apps/migration/model-updates/ipam.md"
+              - UI Component Framework: "development/apps/migration/ui-component-framework.md"
```

Comment:
What do you think about splitting this section to the separate documents like: Introduction, API Documentation, Migration Guide and Examples?